app/serializers/institution_program_profile_serializer.rb

```diff
@@ -11,5 +11,6 @@ class InstitutionProgramProfileSerializer < ActiveModel::Serializer
              :school_locale,
              :tuition_amount,
              :va_bah,
-             :dod_bah
+             :dod_bah,
+             :provider_email_address
```

Comment:
Small nit, can you move provider_email_address next to provider_website?

Reply:
No problem, pushed and updated commit with the re-ordering